packages/store/src/store.ts

```diff
@@ -6,13 +7,14 @@ import { getSelectorFn } from './utils/selector-utils';
 import { InternalStateOperations } from './internal/state-operations';
 import { StateStream } from './internal/state-stream';
 import { enterZone } from './operators/zone';
+import { StateClassStatic } from './internal/internals';
 
 @Injectable()
-export class Store {
+export class Store<T> {
   constructor(
     private _ngZone: NgZone,
-    private _stateStream: StateStream,
-    private _internalStateOperations: InternalStateOperations
+    private _stateStream: StateStream<T>,
```

Comment:
A type `T` des not make sense for the StateStream because the storage structure of the application state is determined by the individual state declarations at run time and a client application should not have any need to map out the structure of the entire application state because of the abstraction provided through selectors.